### PR TITLE
User interface

### PR DESCRIPTION
**Pull request by mjd02**
The code only works on certain browser resolutions. If it's suppose to stay in position for all possible resolutions, please let me know and I'll attempt to fix it.

### CODE REVIEW

**Comment by mjd02** [comment]
Name: Darryl Most

File: project1/css/index.css [hunk 1]
```diff
@@ -62,3 +62,103 @@ h1, h2, h3, h4, h5 {
     letter-spacing: 1px;
     margin-bottom: 10px;
 }
+a{
+	color:gray;
+	text-decoration:none;
+}
+ul{
```
Comment (nomyard) [inline project1/css/index.css]: `ul {

}`

Let's try to keep the spacing around the target element and curly brace consistent throughout the code. The way you want to write code is completely fine, and style suggestions are just that. But when writing code in your chosen style make sure to keep it consistent for nice clean code! :)

File: project1/css/index.css [hunk 2]
```diff
@@ -62,3 +62,103 @@ h1, h2, h3, h4, h5 {
...
+	position:relative;
+	top:8px;
+}
+li {
```
Comment (nomyard) [inline project1/css/index.css]: Same thing here- make sure you are either spacing or not spacing _all_ of the target elements and their respective curly braces.

File: project1/css/index.css [hunk 3]
```diff
@@ -62,3 +62,103 @@ h1, h2, h3, h4, h5 {
...
+	margin: 0 0 0 200px;
+}
+.top-content .text-content p{
+	width:300px;
+	position:absolute;
+	margin: 0 200px 0 0;
+	height:100px;
+	
+}
+
```
Comment (nomyard) [inline project1/css/index.css]: Inconsistent spacing between different blocks of code. All of the code up to this point had no space between blocks. 

Consistency is key!

**Comment by nomyard** [comment]
Hi Darryl- just reviewed your code and have to say that it is really well done. The only issues that I saw were style inconsistencies - which will come with repeated practice. Great job on this! Keep up the good work!

-Ray

**Comment by nomyard** [comment]
Please close this PR when read- thanks!

**Comment by nomyard** [comment]
Hey Darryl- just looked over your code and everything looks great. I like the constant commits and explicit commit messages. That's a great habit to get into. Good to see the consistent style throughout. Like I said before there is no right or wrong way to style your code, and what you've done now is great, but when working in a production environment make sure you adapt your style to the employer's needs. 

Great work! 

-Ray

**Comment by nomyard** [comment]
Please close this PR when read- thanks!

**Pull request closed without merging**